sdk/cosmos/azure-cosmos-spark_3-3_2-12/README.md

```diff
@@ -0,0 +1,146 @@
+# Azure Cosmos DB OLTP Spark 3 connector
+
+## Azure Cosmos DB OLTP Spark 3 connector for Spark 3.3
+**Azure Cosmos DB OLTP Spark connector** provides Apache Spark support for Azure Cosmos DB using 
+the [SQL API][sql_api_query].
+[Azure Cosmos DB][cosmos_introduction] is a globally-distributed database service which allows 
+developers to work with data using a variety of standard APIs, such as SQL, MongoDB, Cassandra, Graph, and Table.
+
+If you have any feedback or ideas on how to improve your experience please let us know here:
+https://github.com/Azure/azure-sdk-for-java/issues/new
+
+### Documentation
+
+- [Getting started](https://aka.ms/azure-cosmos-spark-3-quickstart)
+- [Catalog API](https://aka.ms/azure-cosmos-spark-3-catalog-api)
+- [Configuration Parameter Reference](https://aka.ms/azure-cosmos-spark-3-config)
+
+[//]: # (//TODO: add more sections)
+[//]: # (//TODO: Enable Client Logging)
+[//]: # (//TODO: Examples)
+[//]: # (//TODO: Next steps)
+[//]: # (//TODO: Key concepts)
+[//]: # (//TODO: Azure Cosmos DB Partition)
+[//]: # (//TODO: Troubleshooting)
+
+### Version Compatibility
+
+#### azure-cosmos-spark_3-3_2-12
+| Connector | Supported Spark Versions | Minimum Java Version | Supported Scala Versions | Supported Databricks Runtimes |
+|-----------|--------------------------| -------------------- | -----------------------  |-------------------------------|
+| 4.14.1    | 3.3.0                    | 8                    | 2.12                     | 11.\*                         |
```

Comment:
is the min java version 8 or 11 for spark 3.3?

Reply:
@TheovanKraay  can you please confirm this?

Reply:
I cannot find clear information. However, this doc https://spark.apache.org/docs/latest/ says:

_Spark runs on Java 8/11/17, Scala 2.12/2.13, Python 3.7+ and R 3.5+. **Java 8 prior to version 8u201 support is deprecated as of Spark 3.2.0**. When using the Scala API, it is necessary for applications to use the same version of Scala that Spark was compiled for. For example, when using Scala 2.13, use Spark compiled for 2.13, and compile code/applications for Scala 2.13 as well._

So, I think we should baseline our support at 11?